packages/trader/src/Stores/useTraderStores.tsx

```diff
@@ -20,17 +26,57 @@ type TToastBoxObject = {
     list?: TToastBoxListItem[];
 };
 
+type TContractTypesList = {
+    [key: string]: {
+        name: string;
+        categories: Array<{
+            text: string;
+            value: string;
+        }>;
+    };
+};
+
 type TOverrideTradeStore = Omit<
     TradeStore,
     | 'accumulator_range_list'
+    | 'basis_list'
+    | 'cancellation_range_list'
     | 'contract_purchase_toast_box'
+    | 'contract_types_list'
     | 'clearContractPurchaseToastBox'
+    | 'duration_units_list'
+    | 'duration_min_max'
+    | 'expiry_date'
+    | 'expiry_time'
+    | 'expiry_type'
+    | 'form_components'
+    | 'market_close_times'
+    | 'market_open_times'
+    | 'multiplier_range_list'
     | 'proposal_info'
+    | 'sessions'
+    | 'start_dates_list'
+    | 'start_time'
     | 'ticks_history_stats'
+    | 'trade_types'
 > & {
-    accumulator_range_list?: number[];
+    accumulator_range_list: number[];
+    basis_list: Array<TTextValueStrings>;
     contract_purchase_toast_box: TToastBoxObject;
+    contract_types_list: TContractTypesList;
     clearContractPurchaseToastBox: () => void;
+    duration_units_list: Array<TTextValueStrings>;
+    duration_min_max: {
+        [key: string]: { min: number; max: number };
+    };
+    expiry_date: string | null;
+    expiry_time: string | null;
+    expiry_type: string | null;
+    form_components: string[];
+    market_open_times: string[];
```

Comment:
same goes for `market_close_times`
```suggestion
    market_open_times: moment.Moment[];
```

Otherwise moment methods do no exist on string like an example below

time.minute does not exist on string but on moment it does.

```
const getClosestTime = (time: moment.Moment, interval: number): moment.Moment => {
    return time.minute(Math.ceil(time.minute() / interval) * interval);
};

getClosestTime(market_open_times[i], 5).format('HH:mm');
```

Reply:
@henry-deriv This fields are set as a result of `ContractType.getTradingTimes()` function execution which returns a string for both `market_open_times` and `market_close_times`. Also, according to Moment lib docs, .format() method returns a `string`: https://github.com/moment/moment/blob/000ac1800e620f770f4eb31b5ae908f6167b0ab2/moment.d.ts#L455C5-L455C11 🙂

Reply:
@henry-deriv it has nothing to do with moment, these are taken directly from trading_times API 🙂:
<img width="479" alt="Screenshot 2023-06-26 at 12 21 59 PM" src="https://github.com/kate-deriv/deriv-app/assets/103177211/24fe467a-0c0a-4985-a6de-fd93de54065a">
In the case you mention, getClosestTime() type signature should be changed to the following correct one:
`const getClosestTime = (time: moment.Moment | string, ...`

Reply:
ok i'll change mine to moment.Moment | string